src/XeroPHP/Models/Accounting/TrackingCategory.php

```diff
@@ -262,4 +263,24 @@ public function setOption($value)
         $this->_data['Option'] = $value;
         return $this;
     }
+
+    /**
+     * @return string $value
+     * Return tracking option id
```

Comment:
Think this docblock should just be adjusted to the return type.

```
/**
 * @return string
 */
```

Reply:
Can adjust if needed, but there are other examples, so tried to make this more consistent.

Existing case:
    /**
     * @return string $value
     * Returns selected option name
     */
    public function getOption()